backend/Models/Database/WhaleSpeciesLocations.cs

```diff
@@ -0,0 +1,8 @@
+namespace WhaleSpotting.Models.Database
+{
+    public class WhaleSpeciesLocations
```

Comment:
You don't need this class at all! Entity Framework takes care of the join tables for you - refer to Bookish, if you had a many-to-many relationship somewhere in there

Reply:
Reminder, you need to delete this file